websites/P/ProBot/presence.ts

```diff
@@ -9,34 +9,165 @@ presence.on("UpdateData", () => {
+		else if (document.location.pathname.includes("panel_logs"))
+			presenceData.details = "Viewing Control Panel Logs";
+		else if (document.location.pathname.includes("mod_actions"))
+			presenceData.details = "Viewing Moderation Actions";
+		else if (document.location.pathname.includes("premium")) {
+			if (document.location.pathname.includes("/manage"))
+				presenceData.details = "Managing their Premium";
+			else presenceData.details = "Reading more about Premium";
+		} else if (document.location.pathname.includes("utility")) {
+			presenceData.details = "Changing Module Settings";
+			presenceData.state = "Utility Commands";
+		} else if (document.location.pathname.includes("mod")) {
+			presenceData.details = "Changing Module Settings";
+			presenceData.state = "Moderation Commands";
+		} else if (document.location.pathname.includes("automod")) {
+			presenceData.details = "Changing Module Settings";
+			presenceData.state = "Automod Features";
+		} else if (document.location.pathname.includes("welcomer")) {
+			presenceData.details = "Changing Module Settings";
+			presenceData.state = "Welcome & Goodbye Features";
+		} else if (document.location.pathname.includes("auto_responder")) {
+			presenceData.details = "Changing Module Settings";
+			presenceData.state = "Auto Responder Features";
+		} else if (document.location.pathname.includes("leveling")) {
+			presenceData.details = "Changing Module Settings";
+			presenceData.state = "Leveling System Features";
+		} else if (document.location.pathname.includes("logs")) {
+			presenceData.details = "Changing Module Settings";
+			presenceData.state = "Server Log Features";
+		} else if (document.location.pathname.includes("colors")) {
+			presenceData.details = "Changing Module Settings";
+			presenceData.state = "Color Commands & Features";
+		} else if (document.location.pathname.includes("autoroles")) {
+			presenceData.details = "Changing Module Settings";
+			presenceData.state = "Auto Role Features";
+		} else if (document.location.pathname.includes("reaction_roles")) {
+			presenceData.details = "Changing Module Settings";
+			presenceData.state = "Reaction Roles";
+		} else if (document.location.pathname.includes("music")) {
+			presenceData.details = "Changing Module Settings";
+			presenceData.state = "Music Commands";
+		} else if (document.location.pathname.includes("temp_link")) {
+			presenceData.details = "Changing Module Settings";
+			presenceData.state = "Temp Link Features";
+		} else if (document.location.pathname.includes("voice_online")) {
+			presenceData.details = "Changing Module Settings";
+			presenceData.state = "Voice Online Feature";
+		} else if (document.location.pathname.includes("anti_raid")) {
+			presenceData.details = "Changing Module Settings";
+			presenceData.state = "Anti-Raid Features";
+		} else if (document.location.pathname.includes("protection")) {
+			presenceData.details = "Changing Module Settings";
```

Comment:
is there any way you could make it so you don't have to repeat this 15 times?

Reply:
@ImRodry Those are the module setting pages... I can't do anything about them

Reply:
Surely there is an element you can search for

Reply:
> Surely there is an element you can search for

I've looked up for every element, from header to body, to find something that can fit but there was nothing
The only variables I could use were two somewhere on the body, but those were language dependent and It doesn't make sense if we use them